### ISSUE TITLE
select rules by tag

### ISSUE THREAD

**Issue by itaysk** [aquasecurity/tracee#532]
signatures have a `tags` field. `tracee-rules --rules` should allow the user to select tags of rules in addition to individual rules.
should follow tracee-ebpf's flag conventions e.g -t event / event set

**Comment by yanivagman** [comment]
Using the new rules as events experience, we can have tags treated the same way as sets.
@josedonizetti WDYT?

**Comment by josedonizetti** [comment]
@yanivagman 100%, when creating the events from signatures we can pass all tags as sets https://github.com/aquasecurity/tracee/blob/main/pkg/events/events.go#L84

We can add a specific filter for it, but below it would be 100% the set logic, only an alias for it

**Comment by yanivagman** [comment]
Great!
We should think how we can also choose signatures according to their severity level

**Comment by josedonizetti** [comment]
would it make sense to make [`properties`](https://github.com/aquasecurity/tracee/blob/main/signatures/rego/disk_mount.rego#L12) dynamic? So for every property one could filter with `--trace property. severity=3`? So any signature property can have the `key` used as a filter under `property.*`

**Comment by yanivagman** [comment]
> would it make sense to make [`properties`](https://github.com/aquasecurity/tracee/blob/main/signatures/rego/disk_mount.rego#L12) dynamic? So for every property one could filter with `--trace property. severity=3`? So any signature property can have the `key` used as a filter under `property.*`

Yes, that can work. @idanr1986 @AsafEitani any comments from your side about that?

**Comment by AsafEitani** [comment]
Sounds good to me

**Comment by itaysk** [comment]
feels like the only real use case of this is to filter by severity. [I suggested to promote severity to event metadata](https://github.com/aquasecurity/tracee/issues/2355#issuecomment-1424770772) so it can be filtered like other fields, if that makes sense to you (need to discuss over there), then maybe we don't need to generalize filter by any property

**Comment by yanivagman** [comment]
With the new event structure now in place, severity is a field within the threat section, as well as other fields which may be of interest.
I think we can expand our scope filters to support filtering those fields, and specifically the severity field
